doc/languages-frameworks/go.section.md

```diff
@@ -142,6 +142,7 @@ Many attributes [controlling the build phase](#variables-controlling-the-build-p
 - [`patchFlags`](#var-stdenv-patchFlags)
 - [`postPatch`](#var-stdenv-postPatch)
 - [`preBuild`](#var-stdenv-preBuild)
+- `env`
```

Comment:
is this right ?

Reply:
I tried to look for documentation on the `env` attr, but couldn't find any. I think it's undocumented, which is really surprising.
I may have just missed because it's so hard to grep for, so I'll add it if anyone finds it.

Reply:
yeah, i cant find anything about env either, but this line is just one word `env` is this enough ?

Reply:
also env shouldn't be documented in go section

i am pretty sure its argument to `mkDerivation` of `stdenv`
https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/generic/make-derivation.nix#L527

Reply:
I'm adding it to the list of attributes used in both "main" package and goModule, in this case. The section just above these lines say:

> Many attributes [controlling the build phase](#variables-controlling-the-build-phase) are respected by both `buildGoModule` and `buildGoPackage`. Note that `buildGoModule` reads the following attributes also when building the `vendor/` goModules fixed output derivation as well:

Which is referenced in `pkgs/build-support/go/module.nix` [here](https://github.com/NixOS/nixpkgs/pull/301928/files#diff-2cb526fd81177205f56ed65459e87babbd4799f50f9440f4cff8230ca447641dR76-R79).

I think `env` should be listed here, but correct me if I'm wrong.

Reply:
i see, thank you for clarification. I was thinking about wrong, `env` is the argument to `buildGoModule` which then gets passed to `stdenv.mkDerivation`